src/simulator/src/canvasApi.js

```diff
@@ -107,9 +107,9 @@ export function changeScale(delta, xx, yy, method = 1) {
     if (!embed && !lightMode) {
         findDimensions(globalScope)
         miniMapArea.setup()
-        $('#miniMap').show()
+        document.querySelector('#miniMap').show()
```

Comment:
You can't call show on DOM object, Could you reopen the PR after validating the changes